docs/units/equivalencies.rst

```diff
@@ -291,6 +291,18 @@ observations at high-energy, be it for solar or X-ray astronomy. Example::
     >>> t_k.to(u.eV, equivalencies=u.temperature_energy())  # doctest: +FLOAT_CMP
     <Quantity 86.17332384960955 eV>
 
+Thermodynamic Temperature Equivalency
+-------------------------------------
+
+This equivalency allows conversion between Jy/beam and "thermodynamic
+temperature", :math:`T_{CMB}`, in Kelvins. Example::
```

Comment:
Could we either include the reference here as well or, perhaps better, rewrite the sentence so we can have a link to `thermodynamic_temperature` (and thus people can easily inspect its docstring).

Reply:
I don't want to open a can of worms... But... If you want to include API cross-ref here, shouldn't we do it for all the other ones to be consistent?